Src/Generic/Viewables/BackgroundWithLabel.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  BackgroundLabel.swift
+//  Hedvig
+//
+//  Created by Gustaf Gunér on 2019-02-14.
+//  Copyright © 2019 Hedvig AB. All rights reserved.
+//
+
+import Flow
+import Form
+import Foundation
+import UIKit
+
+struct BackgroundWithLabel {
+    let labelText: DynamicString
+    let backgroundColor: UIColor?
+    let backgroundImage: UIImage?
+    let textColor: UIColor?
+    
+    init(labelText: DynamicString, backgroundColor: UIColor? = .purple, backgroundImage: UIImage? = nil, textColor: UIColor? = .white) {
```

Comment:
Could you put each argument on it's separate row? 🙌

Reply:
✅